torch/__init__.py

```diff
@@ -140,10 +140,10 @@ def set_default_tensor_type(t):
 
     Example::
 
-        >>> torch.tensor([1.2, 3]).dtype  # default is float32
+        >>> torch.tensor([1.2, 3]).dtype    # initial default is float32
```

Comment:
This isn't always true because `torch.tensor([1]).dtype` gives `torch.long`. Maybe say the initial default for floating-point is `float32`?

Also nit: ``torch.float32`` so users don't try to type out ``float32``